pubspec.yaml

```diff
@@ -30,6 +30,9 @@ dependencies:
   cloud_firestore: ^0.14.0+2
   # flutter_svg: ^0.19.0
 
+dependency_overrides:
+  url_launcher_platform_interface: 1.0.8
+
```

Comment:
I needed to add this to the pubspec.yaml file to start the project. ([flutter/flutter#68545](flutter/flutter#68545))